test/IntegrationTests/test_addcustomer.py

```diff
@@ -1,13 +1,19 @@
 import unittest
 import requests
 import logging
+import os
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+dotenv_path = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'primitivechat', 'src', 'backend', '.env')
```

Comment:
Make it dotenv_path = os.path.join(os.path.dirname(__file__), '..', '..', 'src', 'backend', '.env') everywhere

Reply:
I have replaced this line everywhere